seismicupload/upload/update.py

```diff
@@ -0,0 +1,9 @@
+from .specification import parse_specification
+from .storage import upload_to_storage
+from .database import upload_metadata
+
+
+def upload(config, specification):
```

Comment:
Because you wanted it like that for some reason? 

> So I would suggest to and a update.py file with a single function that does the entire upload (I.e. line 63-65)

Who am I to argue? :smile: